armi/settings/fwSettings/tests/test_tightCouplingSettings.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 TerraPower, LLC
```

Comment:
Oh, I believe this is Gregorian calendar.

Reply:
https://github.com/terrapower/armi/pull/1033/commits/fd0b8128a6811b896749fbf889ad7a59b84b3ed6